### ISSUE TITLE
Feature Request - Set Current Without Saving em EEPROM

### ISSUE THREAD

**Issue by soloam** [firstof9/openevse#80]
Back on the old v2 API, there was a option to on the API to set the charging current without saving it into the EEPROM, this would set the current without saving it, and it would restore it on reboot.

This was very useful to use with automatic current algorithms, for example limit the charging current accordingly with the house consumption or solar panel production. Since this varies a lot the current, it would save the EEPROM from several writes.

This was accomplished with the command "$SC <amp> N". Basically the same command, but followed by the "N".

So my idea was if we could have a service that would set the charging current, but setting the "N" so that its send but not saved to the EEPROM. I tested this on my system, and this where the results:

> $SC 15
< $OK 15^04

> $GC
< $OK 6 32 15 15^17

> $SC 10 N
< $OK 10^01

> $GC
< $OK 6 32 10 15^12

Notice that in the second GC we have 10 and 15.

minamps
    min allowed current capacity
hmaxamps
    max hardware allowed current capacity MAX_CURRENT_CAPACITY_Ln
pilotamps
    current capacity advertised by pilot
cmaxamps
    max configured allowed current capacity (saved to EEPROM)

Thank you

**Comment by firstof9** [comment]
Does the old firmware not support PV divert?

**Comment by soloam** [comment]
Yes that option existed on V2 also! Never used it, but I recall seeing that on the old version!

**Comment by firstof9** [comment]
I am pretty sure the current selector for "max current" already accomplishes this task, it sets the pilot to the amps specified.
If you're looking for something that basically feeds PV output into the charger, you'd really want to use the PV divert feature.

**Comment by soloam** [comment]
I don't think I explained correctly, this is not the same! For example I can do the logic to regulate the output accordingly with a lot of things on home assistant! We should avoid to make several writes to memory! This command avoids that, it's the same used to set the max current, but with a N in the end!

**Comment by soloam** [comment]
I don't think I explained correctly, this is not the same! For example I can do the logic to regulate the output accordingly with a lot of things on home assistant! We should avoid to make several writes to memory! This command avoids that, it's the same used to set the max current, but with a N in the end!

**Comment by firstof9** [comment]
I think I see what you're getting at, this would have to be done in the library.
I'll transfer the issue.
